src/client/app/actions/groups.js

```diff
@@ -17,6 +17,14 @@ export const GROUPSUI_CHANGE_SELECTED_GROUPS_PER_GROUP = 'GROUPSUI_CHANGE_SELECT
 export const GROUPSUI_CHANGE_SELECTED_METERS_PER_GROUP = 'GROUPSUI_CHANGE_SELECTED_METERS_PER_GROUP';
 export const GROUPSUI_CHANGE_DISPLAYED_GROUPS = 'GROUPSUI_CHANGE_DISPLAYED_GROUPS';
 
+export const CREATE_NEW_GROUP = 'CREATE_NEW_GROUP';
+export const EDIT_GROUP_NAME = 'EDIT_GROUP_NAME';
+export const ADOPT_CHILD_GROUPS = 'ADOPT_CHILD_GROUPS';
+export const DISOWN_CHILD_GROUPS = 'DISOWN_CHILD_GROUPS';
```

Comment:
I don't think we need both adopt and disown. How about CHANGE_CHILD_GROUPS.

Reply:
Done.